src/Clock.elm

```diff
@@ -0,0 +1,103 @@
+module Clock exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Html exposing (..)
+import Html.Attributes exposing (style)
+import Task
+import Time
+
+
+
+-- MAIN
+
+
+main =
+    Browser.element
+        { init = init
+        , view = view
+        , update = update
+        , subscriptions = subscriptions
+        }
+
+
+
+-- MODEL
+
+
+type alias Model =
+    { zone : Time.Zone
+    , time : Time.Posix
+    }
+
+
+init : () -> ( Model, Cmd Msg )
+init _ =
```

Comment:
Use `always` instead of consuming an unused argument:
```elm
init : ( Model, Cmd Msg )
init =
    ( Model Time.utc <| Time.millisToPosix 0
    , Task.perform AdjustTimeZone Time.here
    )
```
and then in `main`:
```elm
Browser.element
    { init = always init
    ...
```